libs/shared/left-side-menu/src/lib/shared-left-side-menu.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState, useEffect } from "react";
+
+import './shared-left-side-menu.scss';
+
+import {
+  Drawer,
+  List,
+  ListItem,
+  Typography,
+  ListItemText,
+  IconButton
+} from '@material-ui/core';
+
+import { 
+  ChevronLeft as ChevronLeftIcon,
+  Menu as MenuIcon
+ } from '@material-ui/icons';
+
+interface Type {
+  name: string;
+  path: string
+}
+
+export interface SharedLeftSideMenuProps {
+  title: string,
+  menuItemsArray: Array<Type>
+}
+
+export const SharedLeftSideMenu = (props: SharedLeftSideMenuProps) => {
+  const [state, setState] = useState({
+    menuOpen: false,
+  })
+
+  const [menuItems, setMenuItems] = useState([]);
+
+  useEffect(() => {
+    setMenuItems(props.menuItemsArray);
+  }, [props]);
+
+  const toggleDrawer = (open: boolean) => (
+    event: React.KeyboardEvent | React.MouseEvent
+  ) => {
+    setState({ menuOpen: open });
+  };
+
+  const list = () => (
+    <div
+      role="presentation"
+    >
+      <List>
+        {!!menuItems && menuItems.map((item, index) => (
+        <ListItem
+          component='a'
+          button
+          key={index}
+          href={item.path}
+          className="list-item"
+        >
+          <ChevronLeftIcon />
+          <ListItemText
+            primary={item.name}
+            className="list-item-text"
+          />
+        </ListItem>
+        ))}
+      </List>
+    </div>
+  );
+
+  return (
+    <div>
+      <IconButton
+        color="inherit"
+        aria-label="open drawer"
+        onClick={toggleDrawer(true)}
+        // edge="start"
```

Comment:
Remove this comment if you don't use it, but if you will use it, change comment to `// TODO: edge="start"`